monitors/CRMonitor.js

```diff
@@ -135,9 +134,7 @@ class CRMonitor {
       .toBN(collateral)
       .mul(this.web3.utils.toBN(this.web3.utils.toWei("1")))
       .mul(this.web3.utils.toBN(this.web3.utils.toWei("1")))
-      .div(this.web3.utils.toBN(tokensOutstanding).mul(this.web3.utils.toBN(tokenPrice)))
-      .muln(100)
```

Comment:
removed the `muln(100)` to scale consistently with this PRs update. Removed `.toString()` to return a big number.